docker/README.md

```diff
@@ -1,93 +1,154 @@
-# Trino Gateway Docker Image
+# Trino Gateway Docker image
 
-## About the Container
+The Docker image of Trino Gateway is designed for the following use cases:
 
-This Docker image is designed to:
+* Manual usage in front of Trino clusters by mounting in a configuration file
+* Automated usage with an orchestration system like Kubernetes to simplify
+  deployment
 
-* be spun up in front of Trino clusters by mounting in a configuration file
-* simplify deployment into an orchestration system
+## Build requirements
 
-## Quickstart
-
-### Dependencies
-
-This docker build process requires:
+This docker build process requires the following software:
 
 * [Docker Compose V2](https://docs.docker.com/compose/)
 * jq
 
-### Run the Trino Gateway server
+# Building a custom Docker image
+
+Use the following steps to build a Docker image from your local Trino Gateway
+codebase
+
+First, run the Maven build in the project root.
```

Comment:
```suggestion
First, run the Maven build in the project root as guided in [development](../docs/development.md)
```
nit. am not sure how familiar they are with maven but command is not `mvn clean install` but `./mvnw clean install` which might be confusing for first timer in trino

Reply:
Add inline link